pkg/controller/certificaterequests/venafi/venafi.go

```diff
@@ -88,7 +89,20 @@ func (v *Venafi) Sign(ctx context.Context, cr *cmapi.CertificateRequest, issuerO
 
 	duration := apiutil.DefaultCertDuration(cr.Spec.Duration)
 
-	certPem, err := client.Sign(cr.Spec.CSRPEM, duration)
+	var customFields []venafiinternal.CustomField
+	if annotation, exists := cr.GetAnnotations()[cmapi.VenafiCustomFields]; exists && annotation != "" {
+		err := json.Unmarshal([]byte(annotation), &customFields)
```

Comment:
is it okay to assert a pointer to a nil var? I'd guess so else unit tests would fail!

Reply:
It is! saves you a byte in memory :p (GoLand gave me the tip ;) )